src/components/forms/OnboardingProfessionalForm.tsx

```diff
@@ -0,0 +1,109 @@
+"use client";
+import { educationLevels } from "@/shared/onboarding";
+import { formSchema } from "@/shared/onboarding";
+import { FormSchema } from "@/shared/onboarding";
+import { useForm } from "react-hook-form";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { Input, Label, Select, Form } from "@/shared/components";
+
+export default function OnboardingProfessionalForm() {
+  const {
+    register,
+    handleSubmit,
+    watch,
+    formState: { errors },
+  } = useForm<FormSchema>({
+    resolver: zodResolver(formSchema),
+  });
+  const values = watch([
+    "educationLevel",
+    "profissao",
+    "empresaOrganizacao",
+    "githubPortifolio",
+    "linkedin",
+  ]);
+
+  function onSubmit(data: FormSchema) {
+    console.log(data);
+  }
+
+  return (
+    <div className="bg-slate-800 grid">
+      <Form onSubmit={handleSubmit(onSubmit)}>
+        <div className="grid gap-4 border-gray-900/10">
+          <div className="sm:col-span-2">
+            <Label htmlFor="education-level">Formação</Label>
+            <div className="mt-2">
+              <Select
+                defaultValue=""
+                id="education-level"
```

Comment:
I think we can remove all input `id`s, as we already use register for this purpose.